addons/sourcemod/scripting/SurfTimer.sp

```diff
@@ -1939,6 +1939,7 @@ public void OnClientConnected(int client)
 	g_Stage[0][client] = 1;
 	g_bWrcpTimeractivated[client] = false;
 	g_CurrentStage[client] = 1;
+	g_iInBonus[client] = 0;
```

Comment:
Was throwing errors in server console until client reaches a StartZone iirc as this value was being set after they touch a start zone

Reply:
![image](https://user-images.githubusercontent.com/74899888/169239989-31e4ad02-97de-494e-b19c-61cace329993.png)



**N I C E**